src/prommis/uky/uky_flowsheet.py

```diff
@@ -2307,49 +2442,71 @@ def add_costing(m):
     )
 
     # Precipitation costs
-    # 9.2 is UKy Rare Earth Element Precipitation - Polyethylene Tanks
     # 10.1 is UKy Oxalate Precipitation - Polyethylene Tanks
-    reep_pe_tanks_accounts = ["9.2", "10.1"]
-    m.fs.reep_pe_tanks = UnitModelBlock()
-    m.fs.reep_pe_tanks.capacity = Var(initialize=15.04, units=units.gal)
-    m.fs.reep_pe_tanks.capacity.fix()
-    m.fs.reep_pe_tanks.costing = UnitModelCostingBlock(
+    reep_pe_tanks_accounts = ["10.1"]
+    m.fs.precipitator.volume = Var(initialize=15.04, units=units.gal, bounds=(0, None))
+
+    @m.fs.precipitator.Constraint(reep_pe_tanks_accounts)
+    def volume_scaling_constraint(c, k):
+        return m.fs.precipitator.volume == units.convert(
+            REE_costing_params["1"][k]["RP Value"]
+            * units.gal
+            * (
+                m.fs.precipitator.aqueous_inlet.flow_vol[0]
+                / reference_basis_flow["precipitator_solex_aqueous_flow_vol"]
+            ),
+            to_units=units.gal,
+        )
+
+    m.fs.precipitator.costing = UnitModelCostingBlock(
         flowsheet_costing_block=m.fs.costing,
         costing_method=QGESSCostingData.get_REE_costing,
         costing_method_arguments={
             "cost_accounts": reep_pe_tanks_accounts,
-            "scaled_param": m.fs.reep_pe_tanks.capacity,
+            "scaled_param": m.fs.precipitator.volume,
             "source": 1,
             "n_equip": 1,
             "scale_down_parallel_equip": False,
             "CE_index_year": CE_index_year,
         },
     )
 
-    # 9.3 is UKy Rare Earth Element Precipitation - Tank Mixer
-    # 10.2 is UKy Oxalate Precipitation - Tank Mixer
-    reep_tank_mixer_accounts = ["9.3", "10.2"]
-    m.fs.reep_tank_mixers = UnitModelBlock()
-    m.fs.reep_tank_mixers.power = Var(initialize=0.61, units=units.hp)
-    m.fs.reep_tank_mixers.power.fix()
-    m.fs.reep_tank_mixers.costing = UnitModelCostingBlock(
```

Comment:
Same here - why is 9.3 being deleted? There are also other cases below where lines with 2 cost accounts have been trimmed down to one. Is this intentional?